src/Main.java

```diff
@@ -1,5 +1,34 @@
 public class Main {
     public static void main(String[] args) {
+        WaterPokemon piet = new WaterPokemon("Piet", 1, "high", "fish", "water", "Water", true, false, true, false);
```

Comment:
Het is conventie om eerst alle variabelen te declareren en dan pas de method calls te doen. Dus, je zou jan, joop en pieter ook het liefst hier boven hebben staan.